packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -235,5 +235,15 @@
   "reorderItemRight": "Move right",
   "@reorderItemRight": {
     "description": "The audio announcement to move an item in a Reorderable List right in the list when it is oriented horizontally."
+  },
+
+  "expandedIconTapHint": "Collapse",
+  "@expandedIconTapHint": {
+    "description": "The verb infinitive which describes what happens when an expanded ExpandIcon is activated."
```

Comment:
It might be more helpful to describe this translation as describing what happens an expanded ExpandIcon toggle button is pressed. Likewise for `@collapsedIconTapHint`.

Reply:
This one is tricky because it is used in a very specific way to override part of Talkback's hint text. For example:

Value: expand
Talkback: "double tap to <<expand>>"

Value: to expand
Talback: "double tap to <<to expand>>"

So it needs to be phrased very carefully ... and this also depends on how Talkback translates the phrase as well.